platform/net/routes_searcher_unix.go

```diff
@@ -57,7 +61,11 @@ func (s cmdRoutesSearcher) SearchRoutes() ([]Route, error) {
 		if len(routeEntry) == 0 {
 			continue
 		}
-		routes = append(routes, parseRoute(routeEntry))
+		route, err := parseRoute(routeEntry)
+		if err != nil {
+			continue
```

Comment:
could we log that this happened? in case it's unexpected that we couldn't parse the route